pkgs/development/gnuradio-modules/osmosdr/default.nix

```diff
@@ -27,24 +29,30 @@ let
   version = {
     "3.7" = "0.1.5";
     "3.8" = "0.2.3";
+    "3.9" = "0.2.4";
+    "3.10" = "0.2.4";
   }.${gnuradio.versionAttr.major};
   src = fetchgit {
     url = "git://git.osmocom.org/gr-osmosdr";
     rev = "v${version}";
     sha256 = {
       "3.7" = "0bf9bnc1c3c4yqqqgmg3nhygj6rcfmyk6pybi27f7461d2cw1drv";
       "3.8" = "sha256-ZfI8MshhZOdJ1U5FlnZKXsg2Rsvb6oKg943ZVYd/IWo=";
+      "3.9" = "sha256-d0hbiJ44lEu8V4XX7JpZVSTQwwykwKPUfiqetRBI6uI=";
+      "3.10" = "sha256-d0hbiJ44lEu8V4XX7JpZVSTQwwykwKPUfiqetRBI6uI=";
     }.${gnuradio.versionAttr.major};
   };
 in mkDerivation {
   pname = "gr-osmosdr";
   inherit version src;
-  disabledForGRafter = "3.9";
+  disabledForGRafter = "3.11";
 
   outputs = [ "out" "dev" ];
 
   buildInputs = [
+    libsndfile
     log4cpp
+    spdlog
```

Comment:
I think that log4cpp should be used only for gnuradio <=3.9 and spdlog for >= 3.10.
It seems that they have replaced log4cpp in 3.10:
https://www.gnuradio.org/news/2022-01-17-gnuradio-v3.10.0.0-release/

Reply:
Correct.